invoice_test.go

```diff
@@ -37,7 +37,7 @@ var mockInvoice = map[string]any{
 	"total_amount_cents":                      100,
 	"version_number":                          3,
 	"self_billed":                             false,
-	"file_url":                                "https://getlago.com/invoice/file",
+	"file_url":                                "https://getcom/invoice/file",
```

Comment:
I'm assuming this is an unintended search and replace 😄

Reply:
Damn it 🙈

Reply:
Fixed!